ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -272,6 +272,7 @@ type
     | ARRAY '<' type '>'
     | MAP '<' type ',' type '>'
     | STRUCT '<' identifier type (',' identifier type)* '>'
+    | DECIMAL '(' number ',' number ')'
     | baseType ('(' typeParameter (',' typeParameter)* ')')?
```

Comment:
are these parameters going to be removed in the future?

Reply:
Not sure, but that's probably out of scope for this PR.